openapi/squareup/client.bal

```diff
@@ -0,0 +1,2396 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+# Provides API key configurations needed when communicating with a remote HTTP endpoint.
+public type ApiKeysConfig record {|
+    # Represents API key `Authorization`
+    string authorization;
+|};
+
+# Provides Auth configurations needed when communicating with a remote HTTP endpoint.
+public type AuthConfig record {|
+    # Auth Configuration
+    http:BearerTokenConfig|http:OAuth2RefreshTokenGrantConfig|ApiKeysConfig auth;
+|};
+
+# This is a generated connector for [Square Connect API 2.0](https://squareup.com/us/en) OpenAPI Specification.
+@display {label: "SquareUp", iconPath: "icon.png"}
+public isolated client class Client {
+    final http:Client clientEp;
+    final readonly & ApiKeysConfig? apiKeyConfig;
+    # Gets invoked to initialize the `connector`.
+    # The connector initialization requires setting the API credentials. Please create a Squareup account and obtain tokens.
+    #
+    # + authConfig - Configurations used for Authentication 
+    # + clientConfig - The configurations to be used when initializing the `connector` 
+    # + serviceUrl - URL of the target service 
+    # + return - An error if connector initialization failed 
+    public isolated function init(AuthConfig authConfig, http:ClientConfiguration clientConfig =  {}, string serviceUrl = "https://connect.squareup.com") returns error? {
+        if authConfig.auth is ApiKeysConfig {
+            self.apiKeyConfig = (<ApiKeysConfig>authConfig.auth).cloneReadOnly();
+        } else {
+            clientConfig.auth = <http:BearerTokenConfig|http:OAuth2RefreshTokenGrantConfig>authConfig.auth;
+            self.apiKeyConfig = ();
+        }
+        http:Client httpEp = check new (serviceUrl, clientConfig);
+        self.clientEp = httpEp;
+        return;
+    }
+    # CreateMobileAuthorizationCode
```

Comment:
Shall we add spaces between words if possible in all the scenarios below if possible, else it's ok to go with this.

Reply:
Updated as suggested.